packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.vue

```diff
@@ -26,11 +30,16 @@ const { densityClass } = useDensity(props);
 <template>
   <div :class="['onyx-component', 'onyx-empty', densityClass]">
     <slot name="icon">
-      <OnyxIcon :icon="iconCircleX" size="48px" />
+      <OnyxEmptySVG class="onyx-empty__image" />
     </slot>
 
-    <div class="onyx-empty__label onyx-text onyx-truncation-multiline">
-      <slot></slot>
+    <div class="onyx-empty__text-wrapper">
+      <OnyxHeadline is="h3" class="onyx-empty__label onyx-truncation-multiline">
+        <slot></slot>
+      </OnyxHeadline>
+      <p class="onyx-empty__description onyx-truncation-multiline">
```

Comment:
```suggestion
      <p v-if="!!slots.description" class="onyx-empty__description onyx-truncation-multiline">
```

We should not render the p element when no description is set